lib/db/init.test.ts

```diff
@@ -0,0 +1,7 @@
+import { initDB } from "./init";
+
+describe("initializes database", () => {
+    it("initializes the database", () => {
+        expect(initDB()).toBe(true);
```

Comment:
This would be a good place to import `db` and do some assertion to make sure it's working as you expect:
```js
const stmt = db.prepare('SELECT COUNT(*) FROM lockers')
const [numLockers] = stmt.run() // array restructuring
expect(numLockers).toBe(elwLockerCount + ecsLockerCount);
```
Just an example but there are a few other things you could possibly test,